big-practice/src/js/services/api-service.service.js

```diff
@@ -1,19 +1,22 @@
-/**
- * Get by calling API
- */
-const getApi = (url, handleAfterSuccess) => {
-  // GET method implementation
-  const options = {
-    method: 'GET',
-  };
+export class ApiService {
+  /**
+  * Get by calling API
+  */
+  static async getApi(url, handleAfterSuccess) {
```

Comment:
Instead of using Callback function, pls return data on this function

Reply:
updated!